src/FluentResults/Results/Result.cs

```diff
@@ -176,15 +170,24 @@ public async ValueTask<Result> Bind(Func<ValueTask<Result>> action)
         {
             var result = new Result();
             result.WithReasons(Reasons);
-            
-            if (IsSuccess)
-            {
+
+            if (IsSuccess) {
                 var converted = await action();
                 result.WithReasons(converted.Reasons);
             }
 
             return result;
         }
+
+        public static implicit operator Result(Error error)
+        {
+            return Fail(error);
+        }
+
+        public static implicit operator Result(List<Error> errors)
```

Comment:
I hope there is no conflict with the public static implicit operator Result<TValue>(TValue value)
we will see in the tests.